deploy/crds/aws_v1alpha1_awsfederatedrole_customeradmin_cr.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: AWSFederatedRole
+metadata:
+  name: customer-admin
+spec:
+  roleDescription: This is the description of customer-admin
+  # Custom policy definition 
+  awsCustomPolicy:
+    name:  CustomerAdministratorAccess
+    description: Description of CustomerAdministratorAccess
+    # list of statements for the policy
+    awsStatements:
+      - effect: allow
+        action:
+          - '"ec2:AttachVpnGateway"'
```

Comment:
@rafael-azevedo are these all supposed to be surrounded with `'`

Reply:
They are not anymore removing them